contracts/shared/interfaces/ISftRolesRegistry.sol

```diff
@@ -4,23 +4,23 @@ pragma solidity 0.8.1;
 
 import { IERC165 } from './IERC165.sol';
 
-/// @title Semi-Fungible Token Roles
-/// Note: the ERC-165 identifier for this interface is 0x89cb6ab6
+/// @title Semi-Fungible Token Roles Registry Interface
+/// Note: the ERC-165 identifier for this interface is 0x1ec9fef7
```

Comment:
Did we recalculated the ERC-165 identifier?

Reply:
Yes, is the same as the `roles-registry` repository